### DISCUSSION TITLE
*SOLVED* Motion & Hindsight not working?

### DISCUSSION THREAD

**Discussion by RasmusTQ** [gopro/labs#187] [General]
Hi!

I'm trying to get a Hero 11 to do scene-based motion trigger and hindsight, but regardless of my settings, I can't get it to repeat when hindsight is activated. If I deactivate hindsight, it will start and stop recording and repeat flawlessly, but as soon as I activate hindsight, it records once and then stops. :/ I'm trying to do this at 25 fps and 5k, but even trying to drop the resolution down to 1080 it still won't repeat.
Is there anything I can do to remedy this?

Thank you!

(The use case is camera trapping wildlife)

**Comment by dnewman-gpsw** [comment]
Share your command, as I don't know what you are trying.

**Comment by RasmusTQ** [comment]
Thank you for your swift reply David!

Currently, I can't get mVhS1!SM4H2!R to work. It builds up hindsight, triggers once, and then stops the motion detection but continues building hindsight - behaving as "repeat" was set to "off" for motion detection.
I've tried sending the !reset command to the camera as well as a factory reset, and reinstalling the firmware, but still get the same behavior.
mV!SM4H2!R repeats without issue

This is using a full-size hero 11 black on the v2.01.70 Labs firmware, both on internal battery and external USB power.

**Comment by catch-point** [comment]
You need to include `hS0` before the `!R` command to turn hindsight off before repeat.

Try `mVhS1!SM4H2hS0!R`

**Comment by RasmusTQ** [comment]
Thank you so much James! It works flawlessly now. :)

**Comment by Fougl** [comment]
Hi, I cant get this mVhS1!SM4H2hS0!R to work in v2.10.70 firware. 

 Thanks

**Reply by dnewman-gpsw** [reply]
I just tried it on the latest firmware (v2.10.70) and it worked correctly.  What are you seeing?

**Reply by Fougl** [reply]
Thanks for getting back to me. It just goes into hindsight mode without motion detection and stays there
I use QR code generator page to generate the QR code, then I scan it and leave the gopro on the table. Maybe this qr generator page is the problem?

**Reply by dnewman-gpsw** [reply]
Try this:
![image](https://github.com/gopro/labs/assets/10856084/58bf404e-4e93-4051-9a06-c6dfa5a57e76)

**Reply by Fougl** [reply]
Thanks a lot. But no, doesnt work, same as before. I already did factory reset once. What are my options?

**Reply by Fougl** [reply]
This is from the qr_log.txt
![image](https://github.com/gopro/labs/assets/21178929/8de0e42c-25b0-42a6-8347-0e91f9480bd5)

**Reply by Fougl** [reply]
And what i am trying to achieve in the end is this: 
![image](https://github.com/gopro/labs/assets/21178929/6081413c-c296-4706-9f14-46486b25f6d2)

Works nicely, except the motion detection, same as above

**Reply by dnewman-gpsw** [reply]
That qrlog.txt helps, I seeing I'm testing with a version that patches that bug.  True the hs1 can't be used with !SM command are similar in the current release.  I have not found a workaround.

**Reply by Fougl** [reply]
Thanks again. But im not sure what do you mean. Which version are you using so that this command works for you? When can I expect this feature, because I really need it... 

Can I maybe install version v2.01.70 since it seems like it works from the posts above?

**Reply by dnewman-gpsw** [reply]
You can try an older version.  I expect the next Labs release in a couple of months or so.

**Comment by Fougl** [comment]
Seems it works perfectly on the new v2.12.70
Thanks a lot

**Reply by dnewman-gpsw** [reply]
Thanks for testing.  Motion Detection got a number of fixes.

**Comment by Fougl** [comment]
Hi, I tested on Mini, it does not work properly on LABS_MINI11_02_30_70. Does it work on any older firmware. Will mini have the 5 seconds hindsight available as well?

**Reply by dnewman-gpsw** [reply]
Mini doesn't have Hindsight in the main firmware, so there is no way Labs can enable it.
